android/scripts/gfxrecon.py

```diff
@@ -79,6 +79,7 @@ def CreateReplayParser():
     parser.add_argument('--sync', action='store_true', default=False, help='Synchronize after each queue submission with vkQueueWaitIdle (forwarded to replay tool)')
     parser.add_argument('--remove-unsupported', action='store_true', default=False, help='Remove unsupported extensions and features from instance and device creation parameters (forwarded to replay tool)')
     parser.add_argument('--onhb', '--omit-null-hardware-buffers', action='store_true', default=False, help='Omit Vulkan calls that would pass a NULL AHardwareBuffer* (forwarded to replay tool)')
+    parser.add_argument('--use-captured-swapchain-indices', action='store_true', default=False, help='Enable captured swapchain indices.')
```

Comment:
Could this be something more descriptive of what's actually happening, like "Use the captured indices for swapchain images"?

Reply:
fixed. Thanks.